plugins/source/aws/resources/services/s3/bucket_cors_rules.go

```diff
@@ -18,15 +17,8 @@ func bucketCorsRules() *schema.Table {
 		Name:        "aws_s3_bucket_cors_rules",
 		Description: `https://docs.aws.amazon.com/AmazonS3/latest/API/API_CORSRule.html`,
 		Resolver:    fetchS3BucketCorsRules,
-		Transform:   transformers.TransformWithStruct(&types.CORSRule{}),
-		Columns: []schema.Column{
-			client.DefaultAccountIDColumn(false),
-			{
-				Name:     "bucket_arn",
-				Type:     arrow.BinaryTypes.String,
-				Resolver: schema.ParentColumnResolver("arn"),
-			},
-		},
+		Transform:   transformers.TransformWithStruct(&types.CORSRule{}, transformers.WithPrimaryKeys("ID")),
```

Comment:
We need to verify that all of these fields are required fields.

I just checked this one and it is not required. In [the Examples](https://docs.aws.amazon.com/cli/latest/reference/s3api/get-bucket-cors.html) for the AWS CLI they don't even populate:

```
{
    "CORSRules": [
        {
            "AllowedHeaders": [
                "*"
            ],
            "ExposeHeaders": [
                "x-amz-server-side-encryption"
            ],
            "AllowedMethods": [
                "PUT",
                "POST",
                "DELETE"
            ],
            "MaxAgeSeconds": 3000,
            "AllowedOrigins": [
                "http://www.example.com"
            ]
        },
        {
            "AllowedHeaders": [
                "Authorization"
            ],
            "MaxAgeSeconds": 3000,
            "AllowedMethods": [
                "GET"
            ],
            "AllowedOrigins": [
                "*"
            ]
        }
    ]
}
```

Reply:
nice catch!
I'll close this PR for now as noone actually asked to fix those